INFO - 2024-11-26 04:44:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:44:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:44:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:44:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:44:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:44:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:45:25 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-11-26 04:45:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: auth/valid_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` = :username:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:45:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:45:29 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-11-26 04:45:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: HEAD, Route: auth/valid_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` IS NULL
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:45:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:45:33 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(714): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
CRITICAL - 2024-11-26 04:45:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: HEAD, Route: auth/valid_login]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `user`
WHERE `username` IS NULL
 LIMIT 1', [], false)
 3 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(714): CodeIgniter\Model->doFirst()
 5 APPPATH\Controllers\Auth.php(44): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Auth->valid_login()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:45:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:45:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:45:48 --> mysqli_sql_exception: MySQL server has gone away in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *\nFROM `...', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2024-11-26 04:45:48 --> CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away
[Method: GET, Route: monitoring-laporan/data-meteorologi]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 5 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 04:45:48 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL')
 3 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 7 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 04:45:48 --> [Caused by] mysqli_sql_exception: MySQL server has gone away
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL')
 4 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 8 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:45:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:46:00 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-11-26 04:46:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: monitoring-laporan/data-meteorologi]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:46:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:46:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:46:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:46:18 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-11-26 04:46:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: monitoring-laporan/data-meteorologi]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:47:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:47:05 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-11-26 04:47:05 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: monitoring-laporan/data-meteorologi]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:47:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:47:23 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-11-26 04:47:23 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: monitoring-laporan/data-meteorologi]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:47:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:47:36 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-11-26 04:47:36 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: monitoring-laporan/data-meteorologi]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:48:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2024-11-26 04:48:21 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:186
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php(186): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'perekaman_data', 3306, '', 0)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\perekaman_data_cuaca\system\Database\MySQLi\Connection.php:231
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2024-11-26 04:48:21 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: GET, Route: monitoring-laporan/data-meteorologi]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `meteorologi`
WHERE `tanggalutc` IS NULL', [], false)
 3 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
 6 APPPATH\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(null)
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
 8 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 9 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 04:48:21 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Database\Exceptions\DatabaseException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Database\Exceptions\DatabaseException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 8)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Database\Exceptions\DatabaseException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Database\Exceptions\DatabaseException
Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Database\BaseBuilder.php(1644): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\BaseModel.php(676): CodeIgniter\Model->doFindAll(0, 0)
#4 C:\xampp\htdocs\perekaman_data_cuaca\app\Models\MeteorologiModel.php(20): CodeIgniter\BaseModel->findAll()
#5 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\Monitoring.php(73): App\Models\MeteorologiModel->filterByTanggalUTC(NULL)
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\Monitoring->meteorologi()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#11 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#12 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#13 {main}
[Method: GET, Route: monitoring-laporan/data-meteorologi]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-11-26 04:48:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:50:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-11-26 04:50:27 --> [DEPRECATED] strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in APPPATH\Models\MatahariModel.php on line 22.
 1 APPPATH\Models\MatahariModel.php(22): strtotime(null)
 2 APPPATH\Controllers\Monitoring.php(114): App\Models\MatahariModel->filterByBulan(null)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->hujan()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
WARNING - 2024-11-26 04:50:27 --> [DEPRECATED] strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in APPPATH\Models\MatahariModel.php on line 23.
 1 APPPATH\Models\MatahariModel.php(23): strtotime(null)
 2 APPPATH\Controllers\Monitoring.php(114): App\Models\MatahariModel->filterByBulan(null)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->hujan()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:50:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:50:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:50:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:50:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:50:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-11-26 04:50:35 --> [DEPRECATED] strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in APPPATH\Models\MatahariModel.php on line 22.
 1 APPPATH\Models\MatahariModel.php(22): strtotime(null)
 2 APPPATH\Controllers\Monitoring.php(114): App\Models\MatahariModel->filterByBulan(null)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->hujan()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
WARNING - 2024-11-26 04:50:35 --> [DEPRECATED] strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in APPPATH\Models\MatahariModel.php on line 23.
 1 APPPATH\Models\MatahariModel.php(23): strtotime(null)
 2 APPPATH\Controllers\Monitoring.php(114): App\Models\MatahariModel->filterByBulan(null)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->hujan()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:50:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-11-26 04:50:52 --> [DEPRECATED] strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in APPPATH\Models\MatahariModel.php on line 22.
 1 APPPATH\Models\MatahariModel.php(22): strtotime(null)
 2 APPPATH\Controllers\Monitoring.php(155): App\Models\MatahariModel->filterByBulan(null)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->penguapan()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
WARNING - 2024-11-26 04:50:52 --> [DEPRECATED] strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in APPPATH\Models\MatahariModel.php on line 23.
 1 APPPATH\Models\MatahariModel.php(23): strtotime(null)
 2 APPPATH\Controllers\Monitoring.php(155): App\Models\MatahariModel->filterByBulan(null)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->penguapan()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:50:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:51:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
WARNING - 2024-11-26 04:51:01 --> [DEPRECATED] strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in APPPATH\Models\MatahariModel.php on line 22.
 1 APPPATH\Models\MatahariModel.php(22): strtotime(null)
 2 APPPATH\Controllers\Monitoring.php(197): App\Models\MatahariModel->filterByBulan(null)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->matahari()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
WARNING - 2024-11-26 04:51:01 --> [DEPRECATED] strtotime(): Passing null to parameter #1 ($datetime) of type string is deprecated in APPPATH\Models\MatahariModel.php on line 23.
 1 APPPATH\Models\MatahariModel.php(23): strtotime(null)
 2 APPPATH\Controllers\Monitoring.php(197): App\Models\MatahariModel->filterByBulan(null)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Monitoring->matahari()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Monitoring))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:51:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:51:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:51:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-26 04:51:27 --> BadMethodCallException: Call to undefined method App\Models\MeteorologiModel::getData
[Method: GET, Route: report-laporan/data-meteorologi]
in SYSTEMPATH\Model.php on line 932.
 1 APPPATH\Controllers\Report.php(71): CodeIgniter\Model->__call('getData', [])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Report->meteorologi()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Report))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:51:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:52:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-26 04:52:47 --> ErrorException: Undefined variable $tanggal
[Method: GET, Route: report-laporan]
in APPPATH\Views\pages\report\report.php on line 56.
 1 APPPATH\Views\pages\report\report.php(56): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $tanggal', 'C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Views\\pages\\report\\report.php', 56)
 2 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Views\\pages\\report\\report.php')
 3 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('pages/report/report', [], true)
 5 APPPATH\Views\wrapper.php(38): view('pages/report/report')
 6 SYSTEMPATH\View\View.php(234): include('C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Views\\wrapper.php')
 7 SYSTEMPATH\View\View.php(237): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('wrapper', [], true)
 9 APPPATH\Controllers\Report.php(63): view('wrapper', [...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Report->report()
11 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Report))
12 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:53:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:53:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:54:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:54:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-26 04:54:49 --> BadMethodCallException: Call to undefined method App\Models\MeteorologiModel::getData
[Method: GET, Route: report-laporan/data-meteorologi]
in SYSTEMPATH\Model.php on line 932.
 1 APPPATH\Controllers\Report.php(71): CodeIgniter\Model->__call('getData', [])
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Report->meteorologi()
 3 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Report))
 4 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
INFO - 2024-11-26 04:55:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-26 04:55:59 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::extend() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_404.php:1
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\PageNotFoundException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 404, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\PageNotFoundException
Page Not Found
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(981): CodeIgniter\Exceptions\PageNotFoundException::forPageNotFound(NULL)
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->display404errors(Object(CodeIgniter\Exceptions\PageNotFoundException))
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#4 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#6 {main}
[Method: POST, Route: report-laporan/]
in APPPATH\Views\errors\html\error_404.php on line 1.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-26 04:56:24 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::extend() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_404.php:1
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\PageNotFoundException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 404, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\PageNotFoundException
Report-laporan is not a valid controller name
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(981): CodeIgniter\Exceptions\PageNotFoundException::forPageNotFound('Report-laporan ...')
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->display404errors(Object(CodeIgniter\Exceptions\PageNotFoundException))
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#4 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#6 {main}
[Method: POST, Route: report-laporan/]
in APPPATH\Views\errors\html\error_404.php on line 1.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-11-26 04:57:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:58:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 04:58:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:04:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:04:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:04:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:04:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:04:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:04:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:04:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:05:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:06:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:07:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:07:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:07:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:08:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:08:06 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:08:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:08:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:08:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:08:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:08:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:10:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:10:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:11:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:11:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:11:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2024-11-26 05:12:36 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::extend() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_404.php:1
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\Exceptions\PageNotFoundException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 404, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\Exceptions\PageNotFoundException
Controller or its method is not found: Download::1
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(981): CodeIgniter\Exceptions\PageNotFoundException::forPageNotFound('Controller or i...')
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(364): CodeIgniter\CodeIgniter->display404errors(Object(CodeIgniter\Exceptions\PageNotFoundException))
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#4 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#6 {main}
[Method: GET, Route: download/1]
in APPPATH\Views\errors\html\error_404.php on line 1.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-26 05:19:25 --> ErrorException: require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory
[Method: GET, Route: generate-pdf]
in APPPATH\Controllers\PdfController.php on line 13.
 1 APPPATH\Controllers\PdfController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Controllers\\PdfController.php', 13)
 2 APPPATH\Controllers\PdfController.php(13): require_once()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 05:19:25 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory
#0 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:...', 'C:\\xampp\\htdocs...', 13)
#1 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): require_once()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#7 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#9 {main}
[Method: GET, Route: generate-pdf]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-26 05:20:28 --> ErrorException: require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory
[Method: GET, Route: generate-pdf]
in APPPATH\Controllers\PdfController.php on line 13.
 1 APPPATH\Controllers\PdfController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Controllers\\PdfController.php', 13)
 2 APPPATH\Controllers\PdfController.php(13): require_once()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 05:20:28 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory
#0 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:...', 'C:\\xampp\\htdocs...', 13)
#1 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): require_once()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#7 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#9 {main}
[Method: GET, Route: generate-pdf]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-26 05:20:30 --> ErrorException: require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory
[Method: GET, Route: generate-pdf]
in APPPATH\Controllers\PdfController.php on line 13.
 1 APPPATH\Controllers\PdfController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Controllers\\PdfController.php', 13)
 2 APPPATH\Controllers\PdfController.php(13): require_once()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 05:20:30 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory
#0 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:...', 'C:\\xampp\\htdocs...', 13)
#1 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): require_once()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#7 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#9 {main}
[Method: GET, Route: generate-pdf]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-26 05:20:31 --> ErrorException: require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory
[Method: GET, Route: generate-pdf]
in APPPATH\Controllers\PdfController.php on line 13.
 1 APPPATH\Controllers\PdfController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Controllers\\PdfController.php', 13)
 2 APPPATH\Controllers\PdfController.php(13): require_once()
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 9 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 05:20:31 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries/dompdf/autoload.inc.php): Failed to open stream: No such file or directory
#0 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:...', 'C:\\xampp\\htdocs...', 13)
#1 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): require_once()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#7 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#8 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#9 {main}
[Method: GET, Route: generate-pdf]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-26 05:23:54 --> ErrorException: require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries\dompdf/lib/php-font-lib/src/FontLib/Autoloader.php): Failed to open stream: No such file or directory
[Method: GET, Route: generate-pdf]
in APPPATH\Libraries\dompdf\autoload.inc.php on line 20.
 1 APPPATH\Libraries\dompdf\autoload.inc.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Libraries\\dompdf/lib/php-font-lib/src/FontLib/Autoloader.php): Failed to open stream: No such file or directory', 'C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Libraries\\dompdf\\autoload.inc.php', 20)
 2 APPPATH\Libraries\dompdf\autoload.inc.php(20): require_once()
 3 APPPATH\Controllers\PdfController.php(13): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\app\\Libraries\\dompdf\\autoload.inc.php')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 05:23:54 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(ErrorException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(ErrorException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
【Previous Exception】
ErrorException
require_once(C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries\dompdf/lib/php-font-lib/src/FontLib/Autoloader.php): Failed to open stream: No such file or directory
#0 C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries\dompdf\autoload.inc.php(20): CodeIgniter\Debug\Exceptions->errorHandler(2, 'require_once(C:...', 'C:\\xampp\\htdocs...', 20)
#1 C:\xampp\htdocs\perekaman_data_cuaca\app\Libraries\dompdf\autoload.inc.php(20): require_once()
#2 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(13): require_once('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#10 {main}
[Method: GET, Route: generate-pdf]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-26 05:27:48 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "pdf_view.php"
[Method: GET, Route: generate-pdf]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 41.
 1 SYSTEMPATH\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('pdf_view.php')
 2 SYSTEMPATH\Common.php(1171): CodeIgniter\View\View->render('pdf_view', [], true)
 3 APPPATH\Controllers\PdfController.php(22): view('pdf_view', [...])
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
 5 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
 6 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\perekaman_data_cuaca\\public\\index.php')
CRITICAL - 2024-11-26 05:27:48 --> ErrorException: Uncaught Error: Call to undefined method CodeIgniter\Debug\ExceptionHandler::cleanPath() in C:\xampp\htdocs\perekaman_data_cuaca\app\Views\errors\html\error_exception.php:33
Stack trace:
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(266): include()
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\BaseExceptionHandler.php(269): CodeIgniter\Debug\BaseExceptionHandler->CodeIgniter\Debug\{closure}()
#2 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\ExceptionHandler.php(114): CodeIgniter\Debug\BaseExceptionHandler->render(Object(CodeIgniter\View\Exceptions\ViewException), 500, 'C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\Debug\Exceptions.php(163): CodeIgniter\Debug\ExceptionHandler->handle(Object(CodeIgniter\View\Exceptions\ViewException), Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), 500, 1)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\View\Exceptions\ViewException))
#5 {main}
  thrown
【Previous Exception】
CodeIgniter\View\Exceptions\ViewException
Invalid file: "pdf_view.php"
#0 C:\xampp\htdocs\perekaman_data_cuaca\system\View\View.php(222): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('pdf_view.php')
#1 C:\xampp\htdocs\perekaman_data_cuaca\system\Common.php(1171): CodeIgniter\View\View->render('pdf_view', Array, true)
#2 C:\xampp\htdocs\perekaman_data_cuaca\app\Controllers\PdfController.php(22): view('pdf_view', Array)
#3 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(933): App\Controllers\PdfController->generatePdf()
#4 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\PdfController))
#5 C:\xampp\htdocs\perekaman_data_cuaca\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#7 C:\xampp\htdocs\perekaman_data_cuaca\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#8 C:\xampp\htdocs\perekaman_data_cuaca\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#9 C:\xampp\htdocs\perekaman_data_cuaca\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#10 {main}
[Method: GET, Route: generate-pdf]
in APPPATH\Views\errors\html\error_exception.php on line 33.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
INFO - 2024-11-26 05:29:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2024-11-26 05:29:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
